modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -15,17 +15,71 @@ definition {
 				checkboxName = "Enable Accessibility Menu",
 				locator1 = "Checkbox#ANY_CHECKBOX");
 
-			if (IsElementPresent(locator1 = "Button#SAVE")) {
-				Click(locator1 = "Button#SAVE");
-			}
-			else {
-				Click(locator1 = "Button#UPDATE");
-			}
-
-			Alert.viewSuccessMessage();
+			SystemSettings.saveConfiguration();
 		}
 	}
 
+	macro checkOffState {
+		Navigator.openURL();
+
+		KeyPress(
+			locator1 = "//body",
+			value1 = "\TAB");
+
+		KeyPress(
+			key_link = "Skip to Main Content",
+			locator1 = "AccessibilityMenu#QUICK_LINK",
+			value1 = "\TAB");
+
+		KeyPress(
+			key_link = "Open Accessibility Menu",
+			locator1 = "AccessibilityMenu#QUICK_LINK",
+			value1 = "\ENTER");
+
+		AssertElementNotPresent(
+			key_link = "Open Accessibility Menu",
+			locator1 = "AccessibilityMenu#QUICK_LINK");
+	}
+
+	macro checkOnState {
+		Navigator.openURL();
+
+		KeyPress(
+			locator1 = "//body",
+			value1 = "\TAB");
+
+		KeyPress(
+			key_link = "Skip to Main Content",
+			locator1 = "AccessibilityMenu#QUICK_LINK",
+			value1 = "\TAB");
+
+		KeyPress(
+			key_link = "Open Accessibility Menu",
+			locator1 = "AccessibilityMenu#QUICK_LINK",
+			value1 = "\ENTER");
+
+		AssertElementFocused(
+			key_link = "Open Accessibility Menu",
+			locator1 = "AccessibilityMenu#QUICK_LINK");
+	}
+
+	macro deactivateAccessibilityMenu {
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = ${portlet});
+
+		Click(
+			key_itemName = "Accessibility",
+			locator1 = "ListGroupItem#ITEM_TEXT");
+
+		Click(locator1 = "ObjectAdmin#KEBAB_MENU");
+
+		Click(
+			key_option = "Reset Default Values",
+			locator1 = "JobScheduler#KEBAB_MENU_OPTION");
+	}
```

Comment:
Hey @john-co, now we're deactivating the menu by `Reset Default Values`.

![image](https://github.com/liferay-peds/liferay-portal/assets/63858122/5485e5e8-af66-4bbb-af49-a6aac9037a52)

Reply:
I think you had it correct the previous time...deactivateAccessibilityMenu means we uncheck and Save the config to directly disable the accessibility menu.

Reply:
You may want to create a new macro to resetDefaultValues